src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -0,0 +1,1503 @@
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+              <xbrli:context id="Debtors_CY_END">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="core:MaturitiesOrExpirationPeriodsDimension">core:AfterOneYear</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:instant>{{$current_period_end_on}}</xbrli:instant>
+                </xbrli:period>
+              </xbrli:context>
+              {{if $previous_period_start_on}}
+                <xbrli:context id="Debtors_PY_END">
+                  <xbrli:entity>
+                    <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                    <xbrli:segment>
+                      <xbrldi:explicitMember dimension="core:MaturitiesOrExpirationPeriodsDimension">core:AfterOneYear</xbrldi:explicitMember>
+                    </xbrli:segment>
+                  </xbrli:entity>
+                  <xbrli:period>
+                    <xbrli:instant>{{$previous_period_end_on}}</xbrli:instant>
+                  </xbrli:period>
+                </xbrli:context>
+              {{end}}
+              <xbrli:context id="CreditorsWithinOneYear_CY_END">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="core:MaturitiesOrExpirationPeriodsDimension">core:WithinOneYear</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:instant>{{$current_period_end_on}}</xbrli:instant>
+                </xbrli:period>
+              </xbrli:context>
+              {{if $previous_period_start_on}}
+                <xbrli:context id="CreditorsWithinOneYear_PY_END">
+                  <xbrli:entity>
+                    <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                    <xbrli:segment>
+                      <xbrldi:explicitMember dimension="core:MaturitiesOrExpirationPeriodsDimension">core:WithinOneYear</xbrldi:explicitMember>
+                    </xbrli:segment>
+                  </xbrli:entity>
+                  <xbrli:period>
+                    <xbrli:instant>{{$previous_period_end_on}}</xbrli:instant>
+                  </xbrli:period>
+                </xbrli:context>
+              {{end}}
+              <xbrli:context id="CreditorsWithinOneYear_CY">
```

Comment:
`CreditorsWithinOneYear_CY` can be removed

Reply:
06c7096